src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/metric_ops.ts

```diff
@@ -66,6 +66,15 @@ export const formulaOperationDefinitionSchema = schema.object({
    * Reduced time range
    */
   reduced_time_range: schema.maybe(schema.string({ meta: { description: 'Reduced time range' } })),
+  /**
+   * Custom scaling for the entire formula
+   */
+  time_scale: schema.maybe(
+    schema.oneOf(
+      [schema.literal('s'), schema.literal('m'), schema.literal('h'), schema.literal('d')],
+      { meta: { description: 'Time scale' } }
+    )
+  ),
```

Comment:
Can we extract this and reuse it also in `metricOperationSharedSchema`.
I saw the same for the reduced_time_range

Reply:
I'll add it to a follow up where I'll revisit the operations area 👍